server/boot/customRoutes.js

```diff
@@ -7,31 +7,7 @@ module.exports = function(app) {
     var lat = req.query.lat;
     var lng = req.query.lng;
 
-    if (!lat || !lng) {
-      Stores.find(
-        {
-          where: {
-            and: [
-              { isOperating: true },
-              {
-                or: [
-                  { locality: { regex: new RegExp(`.*${location}.*`, "i") } },
-                  { city: { regex: new RegExp(`.*${location}.*`, "i") } },
-                  { address: { regex: new RegExp(`.*${location}.*`, "i") } }
-                ]
-              }
-            ]
-          }
-        },
-        function(err, stores) {
-          if (err) {
-            res.status(400).json(err);
-          } else {
-            res.status(200).json(stores);
-          }
-        }
-      );
-    } else {
+    if (lat && lng) {
```

Comment:
Not merging it for now as ai feel we should have location filter here. would be useful later on

Reply:
Location filter is there. 

Just thought, rather than handling using `if(!lat || !lng)` better we switch if and else case and use `if(lat && lng)`-> just bit clear. 

Reopening PR let me know if any issue.